Bottom: 04ba77b10e3c69eedfa5762fa0eb3eee56db4330
Top:    9a31f45b94fda88d1468a34b541fcc867a5b8c9f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 13:55:23 +0000

Refresh of add-raw-polygons-support-for-t

---

diff --git a/src/global.h b/src/global.h
index 4c6043c..88e64e7 100644
--- a/src/global.h
+++ b/src/global.h
@@ -236,7 +236,7 @@ typedef struct
   void *Element;
 } TextType, *TextTypePtr;
 
-struct			/* holds information about a polygon */
+struct polygon_st			/* holds information about a polygon */
 {
   ANYOBJECTFIELDS;
   Cardinal PointN,		/* number of points in polygon */
